### PR TITLE
DataGrid: add note about editing mode (T1123375)

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The DataGrid uses the DevExtreme [Form](/api-reference/10%20UI%20Components/dxFo
 
 For example, you can specify the [items](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/items.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items') array to reorder editable fields (or "simple items", as they are called in the Form UI component), or organize them in groups and tabs.
 
+[note] We recommend that you wrap all items in groups to ensure that the form fields works properly.
```
Comment (vladaskorohodova) [inline concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md]:
```suggestion
[note] We recommend that you wrap all items in groups to ensure that the form fields work properly.
```
What does "properly" mean? Not clear.

Comment (sergepilipchuk) [reply concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md]: Please, refer to the following ticket: https://supportcenter.devexpress.com/internal/ticket/details/T1123375

File: concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The DataGrid uses the DevExtreme [Form](/api-reference/10%20UI%20Components/dxFo
 
 For example, you can specify the [items](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/items.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items') array to reorder editable fields (or "simple items", as they are called in the Form UI component), or organize them in groups and tabs.
```
Comment (natakazakova) [inline concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md]: If a group is preferable, move it to the first position: group, tabs, or leave items ungrouped.

File: concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md (hunk 1 above)
Comment (natakazakova) [reply concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md]: You can just leave the recommendation without explanation.

```suggestion
[note] The recommended way is to wrap all items in groups.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**